### PR TITLE
[Updated content] Breadcrumb Component

### PR DESCRIPTION
**Pull request by jorytindall**
### :pushpin: Summary

If merged this PR will update the content for the breadcrumb component.

### :hammer_and_wrench: Detailed description

A couple of points that I have specific questions on:

- Is the states section necessary under Specifications? I'm not sure where we landed on this topic.
- Is "Conditionally conformant" the right label to use in the accessibility section?

### :link: External links

Jira ticket: [HDS-1296](https://hashicorp.atlassian.net/browse/HDS-1296)

### CODE REVIEW

**Comment by heatherlarsen** [comment]
@jorytindall 

> Is the states section necessary under Specifications? I'm not sure where we landed on this topic.

If there are interactive states for the component, they should be documented in the "Specifications" tab under the Anatomy section.

File: website/docs/components/breadcrumb/partials/code/component-api.md [hunk 1]
```diff
@@ -1,56 +1,52 @@
...
-    This hook method is called when the component is inserted in the DOM. _Notice: internally we use the "did-insert" modifier from @ember/render-modifiers._
-  </C.Property>
-  <C.Property @name="...attributes">
-    `...attributes` spreading is supported on this component. _Notice: by default an attribute `aria-label="breadcrumbs"` is assigned to the component. If you want to localize it you can override it passing the same attribute with a different value._
-  </C.Property>
+<C.Property @name="itemsCanWrap" @type="boolean">
+This controls if the breadcrumb items can wrap in case they don't fit in the container width.
+</C.Property>
+<C.Property @name="didInsert" @type="function">
+This hook method is called when the component is inserted in the DOM. _Note: internally we use the "did-insert" modifier from `@ember/render-modifiers`._
```
Comment (MelSumner) [inline website/docs/components/breadcrumb/partials/code/component-api.md:16]:
```suggestion
This hook method is called when the component is inserted in the DOM. 

!!! Info

**Note**

Internally, we use the `did-insert` modifier from `@ember/render-modifiers`.

!!!
```

File: website/docs/components/breadcrumb/partials/code/component-api.md [hunk 2]
```diff
@@ -1,56 +1,52 @@
...
+</C.Property>
+<C.Property @name="...attributes">
+`...attributes` spreading is supported on this component. _Note: by default an attribute `aria-label="breadcrumbs"` is assigned to the component. If you want to localize it you can override it passing the same attribute with a different value._
```
Comment (MelSumner) [inline website/docs/components/breadcrumb/partials/code/component-api.md:19]:
```suggestion
This component supports use of `...attributes`.

!!! Info

**Note**

 By default, `aria-label="breadcrumbs"` is defined on the component. If you want to localize it, you can override it by passing the same attribute with a different value.
 
!!!
```

File: website/docs/components/breadcrumb/partials/code/component-api.md [hunk 3]
```diff
@@ -1,56 +1,52 @@
...
-    This controls if this is the last item in the breadcrumb (in which case it doesn't generate a link).
-  </C.Property>
-  <C.Property @name="...attributes">
-    `...attributes` spreading is supported on this component.
-  </C.Property>
+<C.Property @name="text" @type="string">
+The text shown within the item, or "crumb".
+</C.Property>
+<C.Property @name="icon" @type="string">
+Use this parameter to show an icon. Any [Flight](/foundations/icons) icon name is acceptable.
```
Comment (MelSumner) [inline website/docs/components/breadcrumb/partials/code/component-api.md:30]:
```suggestion
Use this parameter to show an icon. Any [icon name](/foundations/icons) is acceptable.
```

File: website/docs/components/breadcrumb/partials/code/component-api.md [hunk 4]
```diff
@@ -1,56 +1,52 @@
...
+</C.Property>
+<C.Property @name="route/models/model/query">
+These are the parameters that are passed down as arguments to the `<LinkTo>` component.
+</C.Property>
+<C.Property @name="current" @type="boolean">
+Determines if an item is the last item in the breadcrumb, in which case it doesn't generate a link.
+</C.Property>
+<C.Property @name="...attributes">
+`...attributes` spreading is supported on this component.
```
Comment (MelSumner) [inline website/docs/components/breadcrumb/partials/code/component-api.md:39]:
```suggestion
This component supports use of `...attributes`.
```

File: website/docs/components/breadcrumb/partials/code/component-api.md [hunk 5]
```diff
@@ -1,56 +1,52 @@
...
-  <C.Property @name="...attributes">
-    `...attributes` spreading is supported on this component.
-  </C.Property>
-</Doc::ComponentApi>
\ No newline at end of file
+<C.Property @name="yield">
+Elements passed as children of this sub-component are yielded to the content of the [Disclosure](../utilities/disclosure) component (used to show/hide the yielded breadcrumb items via a "toggle" button).
+</C.Property>
+<C.Property @name="...attributes">
+`...attributes` spreading is supported on this component.
```
Comment (MelSumner) [inline website/docs/components/breadcrumb/partials/code/component-api.md:50]:
```suggestion
This component supports use of `...attributes`.
```

File: website/docs/components/breadcrumb/partials/code/how-to-use.md [hunk 6]
```diff
@@ -15,30 +15,34 @@ A simple breadcrumb can be invoked using something like this:
...
+    @model={{@cluster.id}}
+  />
   <Hds::Breadcrumb::Item @text="Overview" @current={{true}} />
 </Hds::Breadcrumb>
 ```
 
 #### No wrapping
 
-By default, the breadcrumb allows the item to wrap on multiple lines, if the container width is too small. If you don't want this to happen you have to pass the `itemsCanWrap` parameter as `false`, but keep in mind that the text will be automatically elliptized in this case, to fit the container:
+By default, the breadcrumb allows items to wrap on multiple lines if the container width is too small. If you don't want this to happen, pass `false` to the `itemsCanWrap` parameter, but keep in mind that the text will be automatically replaced with an ellipsis to fit the container:
```
Comment (MelSumner) [inline website/docs/components/breadcrumb/partials/code/how-to-use.md:35]:
```suggestion
By default, the breadcrumb allows items to wrap on multiple lines if the container width is too small. If you don’t want this to happen, pass `false` to the `itemsCanWrap` parameter, but keep in mind that the text will be automatically replaced with an ellipsis to fit the container:
```

File: website/docs/components/breadcrumb/partials/guidelines/guidelines.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+
+### When not to use
+
+- As links outside of the page header.
+  - Within a body of text, use [InlineLink](/components/link/inline/overview)
+  - As a standaloneelement or within a Button Set, use a [StandaloneLink](/components/link/standalone/overview)
+
+!!! Info
+
+The last item in a the breadcrumb should always be the current page.
```
Comment (MelSumner) [inline website/docs/components/breadcrumb/partials/guidelines/guidelines.md:16]:
```suggestion
The last item in a the breadcrumb should always be the current page, and it should not be interactive.
```

File: website/docs/components/breadcrumb/partials/code/component-api.md (hunk 1 above)
Comment (jorytindall) [reply website/docs/components/breadcrumb/partials/code/component-api.md:16]: @MelSumner I did try this initially, but it doesn't seem like a markdown invocation of the banner works inside of the `ComponentApi` component. Unless you got it working, then I'd love to pair with you on it!

**Pull request merged**